Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1484,6 +1484,14 @@ private async Task<AuthenticationResult> AuthenticateNewSessionInternal(Authenti
                 throw new SecurityException("User is not allowed access from this device.");
             }
 
+            int sessionsCount = Sessions.Where(i => string.Equals(i.UserId, user.Id)).ToList().Count;
```

Comment:
```suggestion
            int sessionsCount = Sessions.Count(i => string.Equals(i.UserId, user.Id, StringComparison.OrdinalIgnoreCase));
```

Reply:
These are GUIDs so they *should* have the same case, but no harm.

Reply:
This change actually doesn't work for that exact reason:

```
Session/SessionManager.cs(1487,67): error CS1503: Argument 1: cannot convert from 'System.Guid' to 'string' [/home/vsts/work/1/s/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
```

Reverted the suggestion and leaving as-is.

Reply:
1. They are both Guids. No need to use `string.Equals`. Could just do `i.UserId.Equals(user.Id)`.
2. `ToList()` is a waste of an allocation. Just call `Count()` directly.

Reply:
Confirmed this simpler format works great :+1: